scripts/utils/generate_strings.py

```diff
@@ -89,30 +95,210 @@ def generateStrings():
+        with open(os.path.join(guides_path, guide_filename)) as guide_file:
+            guide_json = json.load(guide_file)
+        if not "id" in guide_json:
+            exit(f"Guide file {guide_filename} does not have an id")
+        if not "title" in guide_json:
+            exit(f"Guide file {guide_filename} does not have a title")
+
+        enum_id = pascalize(f"guide_{guide_json['id']}_title")
+        if enum_id in guide_ids:
+            exit(f"Duplicate id {enum_id} when parsing {guide_filename}")
+        guide_ids.append(enum_id)
+
+        comment = "Title for a guide view"
+        if "comment" in guide_json:
+            comment = guide_json["comment"]
+
+        string_ids.append(
+            {
+                "enum_id": enum_id,
+                "string_id": f"guide.{guide_json['id']}.title",
+                "value": [guide_json["title"]],
+                "comments": [comment],
+            }
+        )
+
+        if not "blocks" in guide_json:
+            exit(f"Guide file {guide_filename} does not have a blocks")
+        for block in guide_json["blocks"]:
+            if not "id" in block:
+                exit(
+                    f"Guide file {guide_filename} does not have an id for one of the blocks"
+                )
+            if not "type" in block:
+                exit(
+                    f"Guide file {guide_filename} does not have a type for block id {block['id']}"
+                )
+            if not "content" in block:
+                exit(
+                    f"Guide file {guide_filename} does not have a content for block id {block['id']}"
+                )
+            enum_id = pascalize(f"guide_{guide_json['id']}_block_{block['id']}")
+            if enum_id in guide_ids:
+                exit(
+                    f"Duplicate id {enum_id} when parsing {guide_filename} - block {block['id']}"
+                )
+            guide_ids.append(enum_id)
+
+            if isinstance(block["content"], list):
+                for subblock in block["content"]:
+                    if not "id" in subblock:
+                        exit(
+                            f"Guide file {guide_filename} does not have an id for one of the subblocks of block {block['id']}"
+                        )
+                    if not "content" in subblock:
+                        exit(
+                            f"Guide file {guide_filename} does not have a content for subblock id {subblock['id']}"
+                        )
+                    enum_id = pascalize(
+                        f"guide_{guide_json['id']}_block_{block['id']}_{subblock['id']}"
+                    )
+                    if enum_id in guide_ids:
+                        exit(
+                            f"Duplicate id {enum_id} when parsing {guide_filename} - subblock {subblock['id']}"
+                        )
+                    guide_ids.append(enum_id)
+
+                    comment = f"A guide block content of type {block['type']}"
```

Comment:
The resulting comment sounds very clunky. What about something like this?

```
                    comment_types = {
                        "text": f"Standard text in a guide block",
                        "title": f"Title in a guide block",
                        "list": f"Bullet list item in a guide block",
                    }
                    if "comment" in subblock:
                        comment = subblock["comment"]
                    else:
                        comment = comment_types.get(block['type'], "")
```

You'd need to fix also line 208.